src/codeu/chat/client/android/app/src/main/java/com/google/codeu/chatme/controller/Controller.java

```diff
@@ -0,0 +1,74 @@
+package com.google.codeu.chatme.controller;
+
+import android.util.Log;
+
+import com.google.codeu.chatme.model.Conversation;
+import com.google.codeu.chatme.model.Message;
+import com.google.codeu.chatme.model.User;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+
+@Deprecated
```

Comment:
Please add an ``@deprecated This class will be removed as and when its functions find a place in our app.`` to the Javadoc comment for this class.